File: Level-1/62.java
class Solution {
    public String solution(int[] food) {
        String answer = "";
        
        String s = "";
        for(int i = 1; i < food.length; i++) {
            int n = food[i] / 2;
            s += String.valueOf(i).repeat(n);
        }
        
        answer = s + "0";
        for(int i = s.length() - 1; i >= 0; i--) {
            answer += s.charAt(i);
        }
        
        return answer;
    }
}

File: Level-1/62.js
function solution(food) {
    let answer = '';
    
    for(let i = 1; i < food.length; i++) {
        food[i] = Math.floor(food[i] / 2);
        answer += String(i).repeat(food[i]);
    }
    
    let reverse = answer.split('').reverse().join('');
    
    return answer + '0' + reverse;
}
